app/src/main/java/org/p2p/wallet/auth/ui/generalerror/GeneralErrorScreenError.kt

```diff
@@ -17,4 +17,10 @@ sealed class GeneralErrorScreenError(
         titleResId = R.string.error_wallet_not_found_title,
         messageResId = R.string.error_wallet_not_found_message
     )
+
+    data class AccountNotFound(
+        val isDeviceShareExists: Boolean,
+        val title: String,
```

Comment:
title and message is set only once and have only one value
you can set title and message inside the GeneralErrorFragment

Reply:
it does not have one value ))

Reply:
you fill in the value depending on the isDeviceShareExists flag
so check it on GeneralErrorScreen